### PR TITLE
Feature/add service roles to clients

### PR DESCRIPTION
**Pull request by oscarfh**
I was using the service and noticed that it would delete my client roles when cleaning up the service account realm roles.

Chart verison update: https://github.com/kiwigrid/helm-charts/pull/274

### CODE REVIEW

File: src/main/java/com/kiwigrid/keycloak/controller/client/ClientController.java [hunk 1]
```diff
@@ -16,10 +18,7 @@
 import javax.ws.rs.core.Response;
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
-import java.util.Base64;
-import java.util.List;
-import java.util.Objects;
-import java.util.Optional;
+import java.util.*;
```
Comment (axdotl) [inline src/main/java/com/kiwigrid/keycloak/controller/client/ClientController.java]: Pls. avoid wild-card imports.

File: src/main/java/com/kiwigrid/keycloak/controller/client/ClientController.java [hunk 2]
```diff
@@ -354,25 +353,25 @@ private void manageServiceAccountRealmRoles(RealmResource realmResource, String
 		var realm = clientResource.getSpec().getRealm();
 		var clientId = clientResource.getSpec().getClientId();
 
+
```
Comment (axdotl) [inline src/main/java/com/kiwigrid/keycloak/controller/client/ClientController.java]: Pls. delete this blank line

**Review by axdotl — CHANGES_REQUESTED** [review]

File: src/main/java/com/kiwigrid/keycloak/controller/client/ClientController.java (hunk 2 above)
Comment (oscarfh) [reply src/main/java/com/kiwigrid/keycloak/controller/client/ClientController.java]: Done

File: src/main/java/com/kiwigrid/keycloak/controller/client/ClientController.java (hunk 1 above)
Comment (oscarfh) [reply src/main/java/com/kiwigrid/keycloak/controller/client/ClientController.java]: Done

**Review by axdotl — APPROVED** [review]

**Pull request merged**